core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/workflow/Progress.java

```diff
@@ -16,8 +16,6 @@
 public interface Progress extends Identifiable, Serializable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)